### PR TITLE
[DOC beta] Ember Component APi - Change to avoid globals

### PR DESCRIPTION
**Pull request by greyhwndz**
@rwjblue: Would be glad to receive feedback on this. Thanks!

### CODE REVIEW

File: packages/ember-views/lib/components/component.js [hunk 1]
```diff
@@ -340,14 +343,14 @@ var Component = View.extend(TargetActionSupport, {
...
-
+    ```handlebars
+    {{! app/templates/application.hbs }}
     {{#foo-bar as |foo|}}
       Hi!
     {{/foo-bar}}
-
-    {{! templates/components/foo-bar.js }}
+    ```
+    ```handlebars
```
Comment (rwjblue) [inline packages/ember-views/lib/components/component.js]: Needs a blank line above and below code fence blocks.

**Comment by rwjblue** [comment]
This looks good, but we need to hold off on changing API doc style completely until we have decided how to represent this in the website.  There are still many globals users and I would like to ensure that they can also use the API docs.

I'll try to remember to bring this up at the next core team meeting so we can decide the best path forward.

**Comment by greyhwndz** [comment]
@rwjblue: Adjusted as you stated. Thanks!

**Comment by greyhwndz** [comment]
@rwjblue: Would you recommend closing this instead?

**Comment by rwjblue** [comment]
No, I'm happy to land this now.  Mind rebasing?

**Comment by locks** [comment]
Discussed offline, will be broken down into individual steps. Thanks @greyhwndz!

**Pull request closed without merging**